index.js

```diff
@@ -1,10 +1,34 @@
 const express = require('express')
 const path = require('path')
 const PORT = process.env.PORT || 5000
+const eventsController = require('./controllers/events');
 
-express()
-  .use(express.static(path.join(__dirname, 'public')))
-  .set('views', path.join(__dirname, 'views'))
-  .set('view engine', 'ejs')
-  .get('/', (req, res) => res.render('pages/index'))
-  .listen(PORT, () => console.log(`Listening on ${ PORT }`))
+//For local testing
+//Guarded include for dotenv as it is not a production dependency
+if(process.env.NODE_ENV!="production"){
+  console.log(`Running locally in ${process.env.NODE_ENV}`);
+  const env = require('dotenv');
+  env.config();
+  if(env.error) throw env.error;
+}
```

Comment:
what does this code do?

Reply:
This requires the dotenv package when you are running this app locally with your keys from your local .env file. It will be ignored when deployed to heroku